django_socio_grpc/tests/grpc_test_utils/fake_grpc.py

```diff
@@ -80,6 +81,35 @@ def __next__(self):
         else:
             return response
 
+    def invocation_metadata(self):
+        return self._invocation_metadata
+
+    def _check_metadata(self, metadata):
+        """
+        Custom method of _BaseFakeContext to be sure tests match reality because grpc metadata should be a tuple of tuple with first element a lower case string and the second a str or bytes value type
+        """
+        for k, value in metadata:
```

Comment:
Yeah I known this causes backward incompatibility but only in test and because of a mistake of us.
gRPC metadata keys should only be lowercase. By not checking it in test we let the possibility of code working in test and not in production code. This also create confusion for users as already seen here: https://github.com/socotecio/django-socio-grpc/issues/209